tests/encodehelp.h

```diff
@@ -71,7 +71,7 @@ static void print_help(const char* app)
     printf("   -N <number of frames to encode(camera default 50), useful for camera>\n");
     printf("   --qp <initial qp> optional\n");
     printf("   --rcmode <CBR|CQP> optional\n");
-    printf("   --ipbmode <0(I frame only ) | 1 (I and P frames) | 2 (I,P,B frames)> optional\n");
+    printf("   --ipbmode <0 (I frame only) | 1 (I and P frames) | N (I,P,B frames, N is the number of B frames, N > 1)> optional\n");
```

Comment:
Changed to ipperiod. Patch submitted.

Best Regards,
Xiaoxia Zhong

From: lizhong1008 [mailto:notifications@github.com]
Sent: Thursday, March 03, 2016 9:37 PM
To: 01org/libyami
Cc: Zhong, Xiaoxia
Subject: Re: [libyami] tests: Add B frame number setting. Add idrinterval for yamitranscode. (#452)


In tests/encodehelp.h<https://github.com/01org/libyami/pull/452#discussion_r54879629>:

> @@ -71,7 +71,7 @@ static void print_help(const char* app)

>      printf("   -N <number of frames to encode(camera default 50), useful for camera>\n");

>      printf("   --qp <initial qp> optional\n");

>      printf("   --rcmode <CBR|CQP> optional\n");

> -    printf("   --ipbmode <0(I frame only ) | 1 (I and P frames) | 2 (I,P,B frames)> optional\n");

> +    printf("   --ipbmode <0 (I frame only) | 1 (I and P frames) | N (I,P,B frames, N is the number of B frames, N > 1)> optional\n");

I prefer to change it to ipperiod, because mode means it's limited.

—
Reply to this email directly or view it on GitHub<https://github.com/01org/libyami/pull/452/files#r54879629>.